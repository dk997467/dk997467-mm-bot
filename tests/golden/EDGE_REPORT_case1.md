EDGE REPORT

| symbol | gross_bps | fees_eff_bps | adverse_bps | slippage_bps | inventory_bps | net_bps | fills | turnover_usd |
|--------|-----------|--------------|-------------|---------------|----------------|---------|-------|---------------|
| BTCUSDT | 0.100000 | 0.100000 | 1.000000 | -0.099001 | 0.009994 | -1.009995 | 3 | 450.150000 |
| ETHUSDT | 0.100000 | 0.100000 | 1.000000 | -0.099001 | 0.004999 | -1.004998 | 3 | 1601.800000 |
| TOTAL | 0.100000 | 0.100000 | 1.000000 | -0.099001 | 0.007497 | -1.007497 | 6 | 2051.950000 |


