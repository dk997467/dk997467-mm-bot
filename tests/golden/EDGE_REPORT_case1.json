{"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"symbols":{"BTCUSDT":{"adverse_bps":16.666667,"fees_eff_bps":-0.100000,"fills":3.000000,"gross_bps":10.000000,"inventory_bps":-0.001112,"net_bps":3.242205,"slippage_bps":-6.656683,"turnover_usd":449.750000},"ETHUSDT":{"adverse_bps":15.000000,"fees_eff_bps":-0.100000,"fills":3.000000,"gross_bps":10.000000,"inventory_bps":-0.002500,"net_bps":8.231666,"slippage_bps":-1.665834,"turnover_usd":2399.600000}},"total":{"adverse_bps":15.833333,"fees_eff_bps":-0.100000,"fills":6.000000,"gross_bps":10.000000,"inventory_bps":-0.001806,"net_bps":5.736935,"slippage_bps":-4.161259,"turnover_usd":2849.350000}}


